dart/lib/src/http_client/failed_request_client.dart

```diff
@@ -165,6 +167,7 @@ class FailedRequestClient extends BaseClient {
       url: urlWithoutQuery,
       queryString: query,
       cookies: sendDefaultPii ? request.headers['Cookie'] : null,
+      // TODO FIXME - this should check [sendDefaultPii], same as DIO integration.
```

Comment:
No need, the default here is `MaxRequestBodySize.never`
So if the user has set something else, this is opt-in for whatever the data is, even if PII.

Reply:
OK, according to that logic, the following should be removed too? https://github.com/getsentry/sentry-dart/blob/main/dio/lib/src/dio_event_processor.dart#L143

Reply:
Mmm good catch, I've missed that, I guess its an oversight on the Dio integration them.
https://docs.sentry.io/platforms/php/configuration/options/#max-request-body-size
Does not mention the`sendDefaultPii` flag , let's keep this TODO here and I will double-check internally.

Reply:
actually, with the following comment that I should remove `body` and replace it with `bodySize`, it's OK anyway, since body size itself is not PII.

Reply:
Yep but `body` still lives in the `Request`, it's not only there in the `Response`.

Reply:
Let's add the `sendDefaultPii` check here for the `body`

Reply:
ok, added the check